packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -583,6 +585,7 @@ export interface VizTooltipOptions {
 export enum TableCellDisplayMode {
   Auto = 'auto',
   BasicGauge = 'basic',
+  Chart = 'chart',
```

Comment:
Seems like this should be called `Sparkline` as opposed to `Chart`. Personally I think if I saw a _Chart_ cell I wouldn't know exactly what it was.

Reply:
@torkelo wdyt? go with `Sparkline`?

Reply:
I would think the word chart or Graph is more understandable/ intuitive than sparkline? Which is a more technical / implementation term no?

Reply:
At least in the software I've used it's been called Sparkline (e.g. Excel, Tableau). Chart sounds like it could be any chart. That being said I'm certainly not the authority on this and happy with whatever name we go with.

Reply:
google, microsoft and wikipedia all agree that a "small inline chart, typically without axes and representing a trend" is a "sparkline". Renamed